.shellcheckrc

```diff
@@ -0,0 +1,21 @@
+# Allow opening any 'source'd file, even if not specified as input
```

Comment:
suggestion (blocking): please add some comments to this file explaining what it is / what uses it